site/docs/api.md

```diff
@@ -172,6 +172,7 @@ Supported predicate expressions are:
 * `in`
 * `notIn`
 * `startsWith`
+* `notStartsWith`
```

Comment:
I added this to the expressions listed in the API docs, though I'm not sure how often we publish the docs. Likely this should not be reflected in the documentation until a version of iceberg is released with `NOT_STARTS_WITH` in it.

If we publish the docs when we publish a new version, then this should be fine to add now. Otherwise, I'll remove it and add a new PR that can be merged in at the appropriate time.

Reply:
Marking this as resolved to make this easier to review as this is not a concern.